_unittest/test_00_EDB.py

```diff
@@ -938,166 +938,6 @@ def test_82d_place_on_bottom_of_lam_with_mold_solder(self):
             chipEdb.close_edb()
             laminateEdb.close_edb()
 
-    def test_82e_place_zoffset_chip(self):
```

Comment:
Why did you remove the Test?

Reply:
@maxcapodi78 I didn't touch anything. I got conflict and merged the main to resolve